strings/names/first_female.txt

```diff
@@ -237,7 +236,6 @@ Erica
 Erika
 Erin
 Ermintrude
-Esmeralda
```

Comment:
Esmeralda is emerald in Spanish but it is also a common name!

Reply:
Even emerald is a normal name